### ISSUE TITLE
Write a proper documentation

### ISSUE THREAD

**Issue by TimothyGillespie** [TimothyGillespie/flashy_images_detection#5]
I planned for it to contain the typical installation and usage guide with examples (from youtube; links, no video in the repo).

Also for it to contain the details and issues I have run into and the implications of the implementation.

**Comment by Korosensei42** [comment]
In principle this is good, but at a later stage you might want to think about how to make the setup as easy as possible for a user. Ideally via a browser plugin.

If you want to go in this direction, you should look up how to publish a browser plugin as a developer.
You might also want to get in touch with the Epilepsy foundation, if you want people to actually use it.

Also it might me a good idea to look by Dietrich Paulus' office at a later stage. This topic might be interesting for him, since he teaches Image Processing. It could even become a fitting bachelor or master thesis with him as a supervisor.

**Comment by TimothyGillespie** [comment]
What your saying is true. If it works well I'd want to make it easily accessible to users and the browser plugin is a great idea on how to do it.

My plan was to keep this repository to simply recognizing these instances, so it can be imported to other projects easily or be used on a server to automatize checking of video/gifs.

I'd imagined the browser plugin to be in a separate repository later on and for this to be more developer focused.

And thanks for the other pointers as well!

**Comment by Korosensei42** [comment]
You are right. From a developer-perspective git is superior. I was thinking with the user in mind, but that will only be important later down the road.
